geniza/urls.py

```diff
@@ -18,14 +18,13 @@
 from django.urls import include, path
 from django.views.generic.base import RedirectView
 
-
 urlpatterns = [
     # redirect homepage to admin site for now
-    path('', RedirectView.as_view(url='admin/')),
```

Comment:
@kmcelwee to prevent it overriding the corpus URLs you could do a `path` of `"/"` (or use `re_path` to specify string terminator if that doesn't work)